android/app/build.gradle

```diff
@@ -23,9 +23,9 @@ android {
     namespace "com.zulip.flutter"
 
     compileSdkVersion flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
```

Comment:
android build: Avoid specifying ndkVersion explicitly

This commit message needs a bit more explanation, so that our future selves and/or successors reading this a couple of years from now when the question of the NDK version comes up again don't have to wonder what we were thinking or sleuth it out.

I'll write a version.